20200804035648_add_explanation_to_profile_fields.rb

```diff
@@ -0,0 +1,5 @@
+class AddExplanationToProfileFields < ActiveRecord::Migration[6.0]
```

Comment:
issue (blocking): @citizen428 oops this file is in the wrong place, it's in the root folder instead of in db/migrate

Reply:
Haha, I had temporarily moved it away when fixing some things locally, then moved it back into the wrong location 🤦 Thanks for spotting this!